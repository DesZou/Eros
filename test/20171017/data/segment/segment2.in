10 10 10
86389 91947 80526 39249 97441 70198 7795 14190 79036 72562 
1 2 9 14
2 1 5 69533
2 4 5 61973
1 3 4 8
2 1 9 76750
2 2 2 53201
1 3 7 96
2 4 7 73989
1 4 5 47
2 6 8 4848
4 5
4 6
2 10
2 9
1 2
2 10
8 10
5 6
3 10
4 4
