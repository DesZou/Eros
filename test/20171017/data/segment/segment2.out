141186.900
193076.733
574768.233
502206.233
139639.667
574768.233
190122.667
122482.483
508270.067
70594.250
